docker/settings.py

```diff
@@ -21,7 +21,7 @@
 AWS_STORAGE_BUCKET_NAME = os.getenv('AWS_STORAGE_BUCKET_NAME', '')
 
 # Settings that control student data provisioning
-INCLUDED_CLASS_CODES = ['1', '2', '3', '4', '5', '8']
-INCLUDED_CAMPUS_CODES = ['0', '1']
-EXCLUDED_MAJOR_CODES = ['0-EMBA', '0-GEMBA']
+INCLUDED_CLASS_CODES = '["1", "2", "3", "4", "5", "8"]'
```

Comment:
Keep these as lists, not strings

Reply:
I kept getting an error that says env variables must be strings